0.6-dev/index.bs

```diff
@@ -0,0 +1,1787 @@
+The "axes" MUST contain 2 or 3 entries of "type:space" and MAY contain one additional entry of "type:time" and MAY contain one
+additional entry of "type:channel" or a null / custom type.  The order of the entries MUST correspond to the order of dimensions
+of the Zarr arrays. In addition, the entries MUST be ordered by "type" where the "time" axis must come first (if present),
+followed by the  "channel" or custom axis (if present) and the axes of type "space".  If there are three spatial axes where two
+correspond to the image plane ("yx") and images are stacked along the other (anisotropic) axis ("z"), the spatial axes SHOULD be
+ordered as "zyx".
+
+Each "multiscales" dictionary MUST contain the field "datasets", which is a list of dictionaries describing the arrays storing
+the individual resolution levels.  Each dictionary in "datasets" MUST contain the field "path", whose value is a string
+containing the path to the Zarr array for this resolution relative to the current Zarr group. The "path"s MUST be ordered from
+largest (i.e. highest resolution) to smallest.  Every Zarr array referred to by a "path" MUST have the same number of dimensions
+and MUST NOT have more than 5 dimensions.  The number of dimensions and order MUST correspond to number and order of "axes".
+
+Each dictionary in "datasets" MUST contain the field "coordinateTransformation", whose value is a dictionary that defines a
+transformation that maps Zarr array coordinates for this resolution level to the default coordinate system (the first entry of
+the "coordinateSystems" array). The transformation is defined according to [[#trafo-md]].  The transformation MUST take as input
+points in the array coordinate system corresponding to the Zarr array at location "path." The value of "input" SHOULD equal the
+value of "path", but implementations should always treat the value of "input" as if it were equal to the value of "path". The
+value of the transformation’s "output" MUST be the name of the default coordinate system.
+
+This transformation SHOULD be one of the following:
+
+* A single scale transformation
+* A sequence transformation containing one scale and one translation transformation.
+
+In these cases, the scale transformation specifies the pixel size in physical units or time duration. If scaling information is
+not available or applicable for one of the axes, the value MUST express the scaling factor between the current resolution and
+the first resolution for the given axis, defaulting to 1.0 if there is no downsampling along the axis. This is strongly
+recommended so that the the "default" coordinate system of the image avoids more complex transformations.
+
+If applications require additional transformations, each "multiscales" dictionary MAY contain the field
+"coordinateTransformations", describing transformations that are applied to all resolution levels in the same manner, after
```

Comment:
ok this seems to answer my [earlier question](https://github.com/ome/ngff/pull/138/files#r2082028698), is the intent to allow the following in zarr metadata?
```
"coordinateTransformation": {...transform stuff}
"coordinateTransformations": [transform stuff]
```

because that's incredibly confusing

Reply:
I agree, though it may just be a byproduct of the draft status of this PR.